packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1924,6 +1932,9 @@ function commitRootImpl(root, renderPriorityLevel) {
       }
     } while (nextEffect !== null);
 
+    // We no longer need to track the active instance fiber
+    focusedInstanceHandle = null;
```

Comment:
FWIW, in this file we do call them fibers. It's just the host config that's intended to mostly stay agnostic.